modules/service/src/main/scala/lucuma/odb/graphql/input/TargetEnvironmentInput.scala

```diff
@@ -10,26 +10,36 @@ import lucuma.odb.data.Nullable
 import lucuma.odb.graphql.binding.TargetIdBinding
 import lucuma.odb.graphql.binding.*
 
-//# Target environment editing and creation parameters
-//input TargetEnvironmentInput {
-//  # The explicitBase field may be unset by assigning a null value, or ignored by skipping it altogether
-//  explicitBase: CoordinatesInput
-//  asterism: [TargetId!]
-//}
-
-final case class TargetEnvironmentInput(
-  explicitBase: Nullable[CoordinatesInput],
-  asterism:     Nullable[List[Target.Id]]
-)
-
-object TargetEnvironmentInput {
-
-  val Binding: Matcher[TargetEnvironmentInput] =
-    ObjectFieldsBinding.rmap {
-      case List(
-        CoordinatesInput.Binding.Nullable("explicitBase", rBase),
-        TargetIdBinding.List.Nullable("asterism", rAsterism)
-      ) => (rBase, rAsterism).parMapN(TargetEnvironmentInput(_, _))
-    }
-
-}
+trait TargetEnvironmentInput:
+  def asterism: Nullable[List[Target.Id]]
+
+object TargetEnvironmentInput:
+
+  final case class Create(
+    explicitBase: Nullable[CoordinatesInput.Create],
+    asterism:     Nullable[List[Target.Id]]
+  ) extends TargetEnvironmentInput
```

Comment:
I'm curious about the `Nullable` in the `Create` as opposed to `Option`.  Maybe it's all the same.

Reply:
Good point, I'll change that.